### ISSUE TITLE
Cacher does not allow access from connected qubes

### ISSUE THREAD

**Issue by unman** [unman/shaker#30]
configuration still retains iptables commands, not recognised in debian-12

**Comment by ben-grande** [comment]
https://github.com/unman/shaker/blob/ffc57008cae7e02ace6078d790af8bcaf7909fc1/cacher/configure.sls#L5-L14

For mail reading:
```
/rw/config/rc.local:
     ...
        /sbin/iptables -I INPUT -p tcp --dport 8082 -j ACCEPT

/rw/config/qubes-firewall-user-script:
    ...
    - text: /sbin/iptables -I INPUT -p tcp --dport 8082 -j ACCEPT
```

Would you accept a PR for:

- Conversion to nftables; and
- Modification to use drop configuration (rc.local.d/*.rc, qubes-firewall.d/*.conf).

The use of drop-in configuration is intended to avoid appending the code multiple times in case the user did modify the files.

**Comment by ben-grande** [comment]
Changed from iptables to nftables in b00d7f5fb533a765a9b6abadc14181a1c3a607f6

Any desire in migrating the files to the drop-in configuration directories?

**Comment by tlaurion** [comment]
@unman ping

**Comment by unman** [comment]
This is fixed in the versions packaged for 4.2
Closing.
